.snippets/code/ethereum-api/ethersjs/balances.js

```diff
@@ -0,0 +1,35 @@
+// Import ethers
+import { ethers } from 'ethers';
+
+// Define network configurations
+const providerRPC = {
+  EvmContainer: {
+    name: 'dancebox-evm-container',
+    rpc: 'https://fraa-dancebox-3001-rpc.a.dancebox.tanssi.network', // Insert your RPC URL here
+    chainId: 5678, // 0x162E in hex,
+  },
+};
+// Create ethers provider
+const provider = new ethers.JsonRpcProvider(providerRPC.EvmContainer.rpc, {
+  chainId: providerRPC.EvmContainer.chainId,
+  name: providerRPC.EvmContainer.name,
+});
+
+// Define addresses
+const addressFrom = 'ADDRESS_FROM_HERE';
+const addressTo = 'ADDRESS_TO_HERE';
```

Comment:
These variable placeholders should be changed to 'INSERT_WHATEVER_VARIABLE_NAME`  so that INSERT is the first word, followed by whatever variable you're describing, and then HERE should *not* be used at the end. This applies to all the variable placeholders in this PR